### PR TITLE
feat(1386): hot-dashboard for recently sold nfts

### LINKED ISSUE #1386: What's hot dashboard
Showing dashboard of NFT collection which recently has made sales

We can probably recycle something from series-insight
Sort by time since the last sale from recent to oldest

verify queries you've created

| series name | buys | total volume | last sale size in KSM | time since last sale |  median time between sales | 
|-|-|-|-|-|-|

### Ref
- #1818 
- #1285 
- #1496 
- #2346

### PR DESCRIPTION
**Pull request by KngZhi**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #1386 

- feat: query for hot dashboard
- feat: show stats data on stats

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [x] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [x] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer?target=DaWoVNNhBTdqJ5iPN3UcaLeamkPpJeGWdrs8LhvX9TQbadc)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

### CODE REVIEW

**Comment by prachi00** [comment]
Do we not have pagination for this feature?

**Comment by KngZhi** [comment]
> Do we not have pagination for this feature?

For now, we do not. Since the datapoint is quite a few.

**Comment by prachi00** [comment]
Screen Shot 2022-05-22 at 8 48 27 PM
so here the volume traded is 2,479.1020 ksm

Screen Shot 2022-05-22 at 8 48 33 PM
but here it shows 304.2851 ksm

**Comment by KngZhi** [comment]
> but here it shows 304.2851 ksm

This only shows one-week data.

> Showing dashboard of NFT collection which recently has made sales [orginal issue]

**Comment by prachi00** [comment]
> > but here it shows 304.2851 ksm
> 
> This only shows one-week sold nfts data. 😄
> 
> > Showing dashboard of NFT collection which recently has made sales [orginal issue]

oh okay, I feel like the title was misleading for me

**Comment by KngZhi** [comment]
Hi @prachi00 which changes are requested?

**Comment by KngZhi** [comment]
@yangwao may I ask what's your opinion about this? the current is for one week recently sold nfts, it would give more data point for the user.

**Comment by prachi00** [comment]
> Hi @prachi00 which changes are requested?

Just a small label change I mentioned above

**Comment by KngZhi** [comment]
> > Hi @prachi00 which changes are requested?
> 
> Just a small label change I mentioned above

All those data are in the one-week range, I am thinking of maybe adding to the title? :)
like, `Recently KUSAMA Sales in 7 days` or `One week`

**Comment by prachi00** [comment]
> Time Since Last

`Time Since Last` should also be `Time Since Last Sale` @KngZhi

**Comment by prachi00** [comment]
> @yangwao the current data range is one week, it would give more data points for the user. The issue asked for daily, but when I tested it, we got very little data for daily range.

@yangwao can you check this once, if all seems fine, I will go ahead and merge it

**Comment by KngZhi** [comment]
@prachi00 I think we can merge now. if @yangwao has any following issue, I can make a pitch for that, no big deal.

**Review by prachi00 — APPROVED** [review]

File: components/hot/HotTable.vue [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+      <b-table-column
+        v-slot="props"
+        cell-class="is-vcentered"
+        label="Last Sale Size(KSM)">
+        {{ props.row.latestSoldSize }}
+      </b-table-column>
+      <b-table-column
+        v-slot="props"
+        cell-class="is-vcentered"
+        label="Time Since Last">
```
[the remark this answers is not in the document]
Comment (KngZhi) [reply components/hot/HotTable.vue]: Okay.

[the remark this answers is not in the document]
Comment (KngZhi) [reply components/hot/HotTable.vue]: already updated.

**Comment by KngZhi** [comment]
Hey @yangwao @prachi00 everything is updated. :)

**Comment by KngZhi** [comment]
@prachi00 or @yangwao could you merge this?

**Comment by yangwao** [comment]
I guess good for me. Hence higher bounty was meant to write a resolver for this to load faster in some cases and extended easier, I'll make follow up issue to make a resolver into Rubick on this issue.

**Comment by yangwao** [comment]
pay 300 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $300 @ 66.94 USD/KSM ~ 4.482 $KSM
🧗 [DaWoVNNhBTdqJ5iPN3UcaLeamkPpJeGWdrs8LhvX9TQbadc](https://kusama.subscan.io/account/DaWoVNNhBTdqJ5iPN3UcaLeamkPpJeGWdrs8LhvX9TQbadc)
🔗 [0x4c40197d92c1a91a33e5d2a1b49e2f98cb1f5eff8eb4ccf7e137e1a55e1da9a7](https://kusama.subscan.io/extrinsic/0x4c40197d92c1a91a33e5d2a1b49e2f98cb1f5eff8eb4ccf7e137e1a55e1da9a7)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**